example/bwmf/proto/bwmf.proto

```diff
@@ -9,7 +9,25 @@ service BlockData {
 }
 
 message Request {
+  uint64 epoch = 1;
```

Comment:
At first, I wanted to use it for a guard to ensure the request is for the t/d shards at the current epoch. Seems it is not necessary.